src/versions.rs

```diff
@@ -11,13 +11,13 @@ pub const HC_SPIN_VERSION: &str = "^0.500.0-dev.0";
 pub const WEB_SDK_VERSION: &str = "^0.6.20-prerelease";
 
 /// crates.io <https://crates.io/crates/hdi/versions>
-pub const HDI_VERSION: &str = "0.6.0-dev.9";
+pub const HDI_VERSION: &str = "0.6.0-dev.13";
 
 /// crates.io <https://crates.io/crates/hdk/versions>
-pub const HDK_VERSION: &str = "0.5.0-dev.12";
+pub const HDK_VERSION: &str = "0.5.0-dev.16";
 
 /// crates.io <https://crates.io/crates/holochain/versions>
-pub const HOLOCHAIN_VERSION: &str = "0.5.0-dev.13";
+pub const HOLOCHAIN_VERSION: &str = "0.5.0-dev.17";
```

Comment:
Intentionally bumping to last week's release?

Holonix isn't ready for this week's release yet but just wondered

Reply:
I meant that Holonix isn't ready for `dev.18`, it was ready for `dev.17` at the end of last Wednesday

Reply:
Ahh sorry for the misunderstanding, I can keep this PR open till `dev.18` is available on holonix